client/src/components/Schedule/SettingsPanel/ManageCsvButtons.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Upload, Row, Col, Tooltip, Button, Form, message } from 'antd';
+import { RcFile } from 'antd/lib/upload';
+import { UploadFile } from 'antd/lib/upload/interface';
+import { DownloadOutlined, UploadOutlined } from '@ant-design/icons';
+import { CourseService } from 'services/course';
+import { parseFiles, uploadResults } from '../utils';
+
+interface ManageCsvButtonsProps {
+  courseId: number;
+  courseService: CourseService;
+  timezone: string;
+  refreshData: () => void;
+}
+
+const ManageCsvButtons: React.FC<ManageCsvButtonsProps> = ({ courseId, courseService, timezone, refreshData }) => {
+  const [form] = Form.useForm();
+  const [fileList, setFileList] = useState<RcFile[]>([]);
+
+  const onRemoveCsvFile = (_: UploadFile<any>) => setFileList([]);
+  const beforeCsvFileUpload = (file: RcFile) => setFileList([...fileList, file]);
+  const exportToCsvFile = () => {
```

Comment:
consider remove state independent files out of rendering

Reply:
But this is depends on `courseId`